woocommerce/woocommerce
# Remove unused WP test install from e2e environment package

### Changes proposed in this Pull Request:

This PR removes the unused WP test install from the e2e environment package.

Closes https://github.com/woocommerce/woocommerce/issues/29388.

### How to test the changes in this Pull Request:

1. Verify CI passes
2. Spin up the test environment locally and verify the tests pass
